docs/articles/documentation/guides/continuous-integration/circleci-and-lambdatest.md

```diff
@@ -0,0 +1,171 @@
+
+In the same manner, copy the *access key* from LambdaTest and create the `LT_ACCESS_KEY` environment variable in CircleCI to store this key.
+
+## Step 5 - Install the LambdaTest Browser Provider and JUnit XML Reporter
+
+Install the following npm packages locally in your project directory:
+
+* [testcafe-browser-provider-lambdatest](https://www.npmjs.com/package/testcafe-browser-provider-lambdatest) – the LambdaTest browser provider that enables TestCafe to run tests in the LambdaTest cloud,
+* [testcafe-reporter-xunit](https://www.npmjs.com/package/testcafe-reporter-xunit) – the JUnit XML reporter that generates test run reports in the XML format. CircleCI can process XML reports and display test results in the UI.
+
+Run the following command to install both packages:
+
+```sh
+npm install --save-dev testcafe-browser-provider-lambdatest testcafe-reporter-xunit
+```
+
+Commit the `package.json` and `package-lock.json` files updated by npm to the `circleci-project-setup` branch and push the changes. These files allow CircleCI to use a fast `npm ci` command to install the dependencies.
+
+## Step 6 - Add the `test` script to package.json
+
+To specify how CircleCI should launch TestCafe, add the `test` script to the `package.json` file. Use the `testcafe` command in the script to run tests on LambdaTest.
+
+```json
+"scripts": {
+  "test": "testcafe \"lambdatest:Chrome@85.0:Windows 10\" tests/**/* -r xunit:/tmp/test-results/res.xml"
+}
+```
+
+After you install the dependencies and specify the `test` script, `package.json` should look as follows:
+
+```json
+{
+  "scripts": {
+    "test": "testcafe \"lambdatest:Chrome@85.0:Windows 10\" tests/**/* -r xunit:/tmp/test-results/res.xml"
+  },
+  "devDependencies": {
+    "testcafe": "*",
+    "testcafe-reporter-xunit": "*",
+    "testcafe-browser-provider-lambdatest": "*"
+  }
+}
+```
+
+Commit it to the `circleci-project-setup` branch and push the changes.
+
+For more information on how to configure a test run with the `testcafe` command, see [Command Line Interface](../../reference/command-line-interface.md).
+
+## Step 7 - View Test Results
+
+Once you commit the `package.json` changes in the previous step, the build is triggered against `circleci-project-setup`. The CircleCI Dashboard shows the results in the pipeline.
+
+Click the job name to see the details.
+
+![View the pipeline in the Dashboard](../../../images/circleci-lambdatest/view-pipeline.png)
+
+The job view displays information about the completed job including the run duration, latest commit, and build log:
+
+![Job details in the Dashboard](../../../images/circleci-lambdatest/view-job.png)
+
+## Conclusion
+
+A testing configuration for CircleCI and LambdaTest is now set up in the `circleci-project-setup` branch. In a real-world project, you would now create a pull request to merge the configuration into the `master` branch.
```

Comment:
Why do you add a conclusion section? If you read a tutorial about how to configure something, you would be able to configure it after the last step. Otherwise, what is the point of the tutorial?

Reply:
For the sake of the second sentence.

I wanted to say that, normally, users would need to create a pull request to `master` but not in this tutorial.

Reply:
What if I rewrite it as:

## Next Steps

A testing configuration for CircleCI and LambdaTest is now set up in the `circleci-project-setup` branch. In a real-world project, you would now create a pull request to merge the configuration into the `master` branch. However, you can skip this step now because this is out of the tutorial's scope.

Reply:
"you would now create a pull request to merge the configuration into the master branch. However, you can skip this step now because this is out of the tutorial's scope." This is a bit strange. Perhaps:

"you can now create a pull request to merge the configuration into the master branch." and leave it there.